### PR TITLE
Introduce posts pagination

### PR DESCRIPTION
**Pull request by adamkudrna**
obrazek
obrazek

### CODE REVIEW

File: partials/pagination.hbs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{#match pages ">" 1}}
+    <nav class="pagination">
+
+        {{#if prev}}
+            <a href="{{page_url prev}}" class="button button--secondary button--small">
+                <span class="spacing-right-1" aria-hidden="true">&larr;</span>
```
Comment (dlouhak) [inline partials/pagination.hbs:6]: What about mobile screens? Will there be enough space to render both buttons including numbers on one line?

I would suggest quick solution and that is visually hide button labels for mobile...

File: index.hbs [hunk 2]
```diff
@@ -22,6 +22,8 @@
                 {{/foreach}}
             </div>
 
+            {{pagination}}
```
Comment (dlouhak) [inline index.hbs:25]: This template is for other pages than `/articles/` as well?

**Review by dlouhak — COMMENTED** [review]
Still struggling with the fact that HTML entities are not vertically aligned even though they are technically inserted properly. 🙈

File: index.hbs (hunk 2 above)
Comment (adamkudrna) [reply index.hbs:25]: This is actually a shorthand for calling the pagination template and filling it with data (if I get it right). Calling just the template itself (`{{> "pagination"}}`) shouldn't work.

https://ghost.org/docs/themes/helpers/pagination/

If you mean the `index.hbs` file, currently it's a template for all post listings: articles, categories, even newsletters…

File: partials/pagination.hbs (hunk 1 above)
Comment (adamkudrna) [reply partials/pagination.hbs:6]: Good point! Fixed:

obrazek

**Pull request merged**